common/commands/config.go

```diff
@@ -396,6 +396,9 @@ func Export(serverName string) error {
 	if err != nil {
 		return err
 	}
+	if serverDetails.ServerId == "" {
+		return errorutils.CheckErrorf("couldn't find default server")
```

Comment:
couldn't find default server
-->
Cannot export config, becuase it is empty. Run 'jf c add' and then export again